### PR TITLE
🧪 TESTS: Add GFM spec test

### PR DESCRIPTION
**Pull request by hukkinj1**
Based on https://github.com/executablebooks/mdformat-tables/pull/7 not the master branch.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -34,7 +34,6 @@ include = []
 exclude = [".github/", "tests/"]
 
 [tool.isort]
-skip = ["venv"]
```
Comment (hukkinj1) [inline pyproject.toml:37]: It seems isort behavior has changed regarding this conf. Previously it was helpful, now it just overrides the default skip, and doesn't even support folders (there's `skip_glob` I think for that).

File: tests/test_spec.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+)
+def test_spec(entry):
+    """Test mdformat-tables against the GFM spec.
+
+    Test that:
+    1. Markdown AST is the same before and after 1 pass of formatting
+    2. Markdown after 1st pass and 2nd pass of formatting are equal
+    """
+    if entry["name"] == "200":
+        pytest.xfail("Need to fix this in markdown-it-py!")
```
Comment (chrisjsewell) [inline tests/test_spec.py:27]: I assume this is fixed in the develop branch

**Review by chrisjsewell — APPROVED** [review]
👍

File: tests/test_spec.py (hunk 2 above)
Comment (hukkinj1) [reply tests/test_spec.py:27]: :+1: correct

**Pull request merged**